chat-plugins/roomsettings.js

```diff
@@ -247,7 +247,7 @@ exports.commands = {
 			Rooms.global.writeChatRoomData();
 		}
 	},
-	modchathelp: ["/modchat [off/autoconfirmed/+/%/@/*/#/&/~] - Set the level of moderated chat. Requires: *, @ for off/autoconfirmed/+ options, # & ~ for all the options"],
+	modchathelp: ["/modchat [off/autoconfirmed/+/%/@/*/\u2606/#/&/~] - Set the level of moderated chat. Requires: *, @, \u2606 for off/autoconfirmed/+ options, # for off/autoconfirmed/+/%/@/*/\u2606/# options, & ~ for all the options"],
```

Comment:
We definitely don't care about the fact that `#` can't set `modjoin &` or something. That's a tiny detail that doesn't belong in help text.